libraries/scripting/jvm-host-test/test/kotlin/script/experimental/jvmhost/test/LegacyReplTest.kt

```diff
@@ -44,6 +44,13 @@ class LegacyReplTest : TestCase() {
         }
     }
 
+    fun testReplErrors2() {
+        LegacyTestRepl().use { repl ->
+            val res = repl.compileAndEval(repl.nextCodeLine("data class Q(val x: Int, val: String)"))
```

Comment:
There is an error, `val` should be followed by identifier. But classes are successfully compiled.